blocks/library/list/editor.scss

```diff
@@ -2,6 +2,5 @@
 .blocks-list .blocks-editable__tinymce ul,
 .blocks-list .blocks-editable__tinymce ol {
 	padding-left: 1.3em;
-	list-style-position: inside;
-	margin-left: 0;
+	margin-left: 1.3em;
```

Comment:
I believe the "inside" was added for some toolbar issues. cc @gziolo

Reply:
It was added here: https://github.com/WordPress/gutenberg/commit/1c53b02039be3d318350dfa08badfe90fd487af8